### DISCUSSION TITLE
i fixed it now but if u are wondering why u cant start a github codespace this is why

### DISCUSSION THREAD

**Discussion by RandomGamingIn144p** [community/community#75481] [Codespaces]
### Select Topic Area

Bug

### Body

When I try to start the github codespace it just gives the error The workbench failed to connect to the server (Error: Connection error: Unauthorized client refused) and An unexpected error occurred that requires a reload of this page.
reloading doesnt fix it

nvm its because it doesnty work on firefox

**Comment by muffinbyte404** [comment]
did u got it fixed ?
